src/picterra/client.py

```diff
@@ -204,6 +206,60 @@ def upload_raster(self, filename: str, name: str, folder_id=None,
         self._wait_until_operation_completes(resp.json())
         return raster_id
 
+    def upload_remote_raster(
+        self, server_type: str, server_url: str, spatial_resolution_meters: float, footprint: dict,
+        credentials: Optional[str] = None, name: Optional[str] = None,
+        folder_id: Optional[str] = None, captured_at: Optional[str] = None
+    ) -> str:
+        """
+        Upload a remote raster to picterra (WMS or XYZ imagery server sources).
+
+        Args:
+            server_type (str): One of ['wms', 'xyz']
+            server_url (str): URL for the imagery server
+            spatial_resolution_meters (float): GSD of the image we want to import
+            footprint (dict): GeoJSON Polygon containing the boundaries of the remote raster
+            credentials (optional, str): Credentials for the imagery server in the
+                "user:password" format, if needed
+            name (optional, str): Name the raster will take
+            folder_id (optional, str): Identifier of the folder this raster will belongs to
```

Comment:
Perhaps mention here that if that's not provided, we'll put it in the `Picterra API Project` folder.

Reply:
Checked (also updated the docstring of `upload_raster` too)